dspace/modules/api/src/main/java/org/datadryad/rest/models/Manuscript.java

```diff
@@ -189,13 +186,10 @@ public Manuscript() {} // JAXB needs this
 
     public Manuscript(DryadJournalConcept journalConcept) {
         this.journalConcept = journalConcept;
-        this.setOrganization(new Organization(journalConcept));
     }
 
     public Manuscript(LegacyManuscript legacyManuscript) {
         DryadJournalConcept journalConcept = JournalUtils.getJournalConceptByJournalID(legacyManuscript.Journal_Code);
-        this.setJournalConcept(journalConcept);
```

Comment:
Why was this line removed? I don't see any other place in this method where this.journalConcept is set.